app/Resources/views/membership/choose_nearby_committee.html.twig

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html.twig' %}
+
+{% block banner '' %}
+
+{% block title 'Suivre des comités' %}
+
+{% block content %}
+  <section class="l__wrapper--medium">
+    <div class="membership_choose_nearby_committee__form">
+      <h1>Pour terminer</h1>
+      <h4>Suivez l'actualité des commités locaux près de chez vous.<br>C'est le cœur du mouvement En Marche !</h4>
+
+      {{ form_start(form) }}
+
+      {{ form_row(form.committees) }}
```

Comment:
Please indent the lines between `form_start` and `form_end` as you would do with a regular `<form>`tag.